openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -343,6 +343,75 @@ Note: The issuer MAY decide to support both options. In which case, it is at the
 
 A Credential Format Profile for Credentials complying with IETF SD-JWT VCs [@!I-D.ietf-oauth-sd-jwt-vc] is defined in Annex A.3 of [@!OIDF.OID4VCI] and Annex A.4 of [@!OIDF.OID4VP].
 
+## SD-JWT VC Data Model (SD-JWT VCDM)
+
+SD-JWT VCDM is a data model that follows IETF SD-JWT VC [@!I-D.ietf-oauth-sd-jwt-vc], but allows the usage of [@!W3C.VCDM1.1] or [@!W3C.VCDM2.0]. When IETF SD-JWT VC is mentioned in this specification, SD-JWT VCDM define in this section MAY be used.
+
+For backward compatibility with JWT processors, the following registered JWT claims MUST be used, instead of their respective counterpart properties in [@!W3C.VCDM1.1] or [@!W3C.VCDM2.0]:
+
+* `exp` Claim MUST represent the `expirationDate` property, encoded as a UNIX timestamp (NumericDate).
```

Comment:
```suggestion
* `expirationDate` property MUST NOT be used. It MUST be ignored if present. It is represented by `exp` Claim encoded as a UNIX timestamp (NumericDate).
```

Reply:
this is not a transformation logic

Reply:
Official claim is called expiry_date
https://eur-lex.europa.eu/eli/reg_impl/2024/2977/oj